src/instructlab/model/train.py

```diff
@@ -164,8 +282,57 @@ def get_files(pattern):
         shutil.copy(train_files[0], train_file)
         shutil.copy(test_files[0], test_file)
 
-    # pylint: disable=import-outside-toplevel
-    if not utils.is_macos_with_m_chip():
+    # if macos, preserve that path
+    if utils.is_macos_with_m_chip():
+        # Local
+        from ..train.lora_mlx.convert import convert_between_mlx_and_pytorch
+        from ..train.lora_mlx.lora import load_and_train
+        from ..train.lora_mlx.make_data import make_data
+
+        if not skip_preprocessing:
+            try:
+                make_data(data_dir=data_path)
+            except FileNotFoundError as exc:
+                click.secho(
+                    f"Could not read from data directory: {exc}",
+                    fg="red",
+                )
+                raise click.exceptions.Exit(1)
+
+        # NOTE we can skip this if we have a way ship MLX
+        # PyTorch safetensors to MLX safetensors
+        model_dir_local = model_path.replace("/", "-")
+        model_dir_mlx = f"{model_dir_local}-mlx"
+        model_dir_mlx_quantized = f"{model_dir_local}-mlx-q"
+
+        if skip_quantize:
+            dest_model_dir = model_dir_mlx
+            quantize_arg = False
+        else:
+            dest_model_dir = model_dir_mlx_quantized
+            quantize_arg = True
+
+        # Downloading PyTorch SafeTensor and Converting to MLX SafeTensor
+        convert_between_mlx_and_pytorch(
+            hf_path=model_path,
+            mlx_path=dest_model_dir,
+            quantize=quantize_arg,
+            local=local,
+        )
+
+        adapter_file_path = f"{dest_model_dir}/adapters.npz"
+        # train the model with LoRA
+
+        load_and_train(
+            model=dest_model_dir,
+            train=True,
+            data=data_path,
+            adapter_file=adapter_file_path,
+            iters=iters,
+            save_every=10,
+            steps_per_eval=10,
+        )
```

Comment:
```suggestion

        # train the model with LoRA
        load_and_train(
            model=dest_model_dir,
            train=True,
            data=data_path,
            adapter_file=adapter_file_path,
            iters=iters,
            save_every=10,
            steps_per_eval=10,
        )
```

Reply:
nit

Reply:
Easy to implement, code looks less sloppy

Reply:
alright, done